Atala PRISM/2.8.md

```diff
@@ -0,0 +1,44 @@
+# Atala PRISM v2.8
+
+December, 2023 release of Atala PRISM.
+
+## Components
+
+### Platform
+
+* [Open Enterprise Agent v1.21.1](https://github.com/hyperledger-labs/open-enterprise-agent/releases/tag/prism-agent-v1.21.1)
+* [PRISM Node v2.2.1](https://github.com/input-output-hk/atala-prism/releases/tag/v2.2.1)
+* [PRISM Mediator v0.10.2](https://github.com/input-output-hk/atala-prism-mediator/releases/tag/prism-mediator-v0.10.2)
+
+### SDKs
+
+* [Apollo Crypto Library v1.1.1](https://github.com/input-output-hk/atala-prism-apollo/releases/tag/v1.1.1)
+* [Wallet SDK Swift v3.6.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-swift/releases/tag/3.6.0)
+* [Wallet SDK TypeScript v3.1.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts/releases/tag/v3.1.0)
+* [Wallet SDK KMM v2.6.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-kmm/releases/tag/v2.6.0)
+
+### Documentation
+
+* [Documentation Portal v1.69.0](https://github.com/input-output-hk/atala-prism-docs/releases/tag/v1.69.0)
+
+## Changelog
+
+**Updated components:**
+
+- Open Enterprise Agent: v1.19.1 -> v1.21.0
+- Documentation Portal: v1.65.0 -> v1.69.0
+- Apollo Crypto Library: no changes
+- Wallet SDK KMM: no changes
+- Wallet SDK TypeScript: no changes
+- PRISM Mediator: no changes
+- Wallet SDK Swift: no changes
+- PRISM Node: no changes
+
+**New Features:**
+
+- **Open Enterprise Agent**:
+  - Infrastructure changes that allow to configure resources and parameters of the Open Enterprise Agent in Cloud k8s environments.
+- **Documentaiton Portal**:
+  - Removed out-of-date inftastructure documentation.
+  - Fixed typos in Keycloak configuration documentation.
```

Comment:
We can mention that:
- Quick Start Guide is complete and up to date.

Reply:
Thanks! Fixed.